crates/release_plz/src/args/release.rs

```diff
@@ -26,32 +26,48 @@ pub struct Release {
     /// Perform all checks without uploading.
     #[clap(long)]
     pub dry_run: bool,
-    /// Publish GitHub release for the created git tag. (Experimental: we will probably remove this flag)
+    /// Publish GitHub/Gitea release for the created git tag. (Experimental: we will probably remove this flag)
     #[clap(long)]
     pub git_release: bool,
-    /// GitHub repository url.
+    /// GitHub/Gitea repository url.
     #[clap(long, value_parser = NonEmptyStringValueParser::new())]
     pub repo_url: Option<String>,
-    /// Git token used to publish the GitHub release.
+    /// Git token used to publish the GitHub/Gitea release.
     #[clap(long, value_parser = NonEmptyStringValueParser::new())]
     pub git_token: Option<String>,
+    /// Git Backend (GitHub or Gitea).
+    #[clap(long, value_enum, default_value_t = GitBackendKind::Github)]
+    pub git_backend: GitBackendKind,
 }
 
 impl TryFrom<Release> for ReleaseRequest {
     type Error = anyhow::Error;
 
     fn try_from(r: Release) -> Result<Self, Self::Error> {
+        let token = r.git_token.clone().map(SecretString::from).unwrap();
```

Comment:
```suggestion
        let token = r.git_token.clone().map(SecretString::from).context("git_token is required for git_release")?;
```
It's better to avoid unwrapping when the error depends on user input